views/admin/inc/live_chat.tpl

```diff
@@ -0,0 +1,77 @@
+<style>
```

Comment:
Larger blocks of CSS should be put into an own file, see https://docs.oxid-esales.com/developer/en/6.0/modules/certification/software_quality.html#using-css-and-including-css-files